src/interaction/interactiveTarget.js

```diff
@@ -29,7 +31,12 @@ module.exports = {
     /**
      * @todo Needs docs.
      */
-    defaultCursor: 'pointer',
+    defaultCursor: 'pointer',    
+    
+    /**
+     * The threshold in px used to check if a tap occured on the interactive object.
+     */
+    tapThreshold: 8,
```

Comment:
I like the idea of this, but it should be called "radius" not "threshold" to be clear about what it does.